.vscode/tasks.json

```diff
@@ -1,120 +1,137 @@
-            "label": "CompileExtension",
-            "command": "dotnet",
-            "args": [
-                "build"
-            ],
-            "options": {
-                "cwd": "src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/"
-            },
-            "problemMatcher": "$tsc-watch",
-            "presentation": {
-                "reveal": "silent"
-            },
-            "dependsOn": [
-              "CompileLibrary"
-            ]
-        },
-        {
-            "label": "CompileUnitTests",
-            "command": "dotnet",
-            "args": [
-                "build"
-            ],
-            "options": {
-                "cwd": "src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.Test/"
-            },
-            "problemMatcher": "$tsc-watch",
-            "presentation": {
-                "reveal": "silent"
-            }
-        },
-        {
-            "label": "CompileGrammarTests",
-            "command": "dotnet",
-            "args": [
-                "build"
-            ],
-            "options": {
-                "cwd": "src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.Grammar.Test/"
-            },
-            "problemMatcher": "$tsc-watch",
-            "presentation": {
-                "reveal": "silent"
-            }
-        },
-        {
-            "label": "CompileFunctionalTest",
-            "command": "dotnet",
-            "args": [
-                "build"
-            ],
-            "options": {
-                "cwd": "src/Razor/test/VSCode.FunctionalTest/"
-            },
-            "problemMatcher": "$tsc-watch",
-            "presentation": {
-                "reveal": "silent"
-            },
-            "dependsOn": [
-                "CompileExtension"
-            ]
-        },
-    ]
+  "version": "2.0.0",
+  "tasks": [
+      {
+          "label": "CompileTextMate",
+          "type": "npm",
+          "script": "compile:TextMate",
+          "problemMatcher": ["$eslint-stylish"],
+          "options": {
+              "cwd": "${workspaceFolder}/src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/"
+          }
+      },
+      {
+          "label": "CompilePackage",
+          "command": "dotnet",
+          "args": [
+              "build"
+          ],
+          "options": {
+              "cwd": "${workspaceFolder}/src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/"
+          },
+          "problemMatcher": "$tsc-watch",
+          "presentation": {
+              "reveal": "silent"
+          }
+      },
+      {
+          "label": "CompileLibrary",
+          "command": "dotnet",
+          "args": [
+              "build"
+          ],
+          "options": {
+              "cwd": "${workspaceFolder}/src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/"
+          },
+          "problemMatcher": "$tsc-watch",
+          "presentation": {
+              "reveal": "silent"
+          },
+          "dependsOn": [
+              "ActiveDotNet"
```

Comment:
Hmm, why does it need to activate dotnet?

Reply:
When working on the aspnetcore repo locally, you'd do something like this.

```
$ source activate.sh
$ code .
```

The local dotnet runtime is activated in the VS Code session you start.

However, if you're using the dev container, you're starting off on a fresh shell without the activated environment which causes build failures when you try to run the extension. This ensures that you're running in the activated environment even if you're inside a dev container.

Reply:
Totally makes sense. Thanks for the clarification!